features/step_definitions/fixture_steps.ts

```diff
@@ -6,13 +6,15 @@ import {
 } from '../support/formatter_output_helpers'
 import fs from 'mz/fs'
 import path from 'path'
+import { messages } from 'cucumber-messages'
+import Envelope = messages.Envelope
```

Comment:
I'm a bit torn on this. I understand the idea of using this to dry up code usage but also don't like the idea of having a ton of imports up here as we list all the imports we are using (I think one file has at least 5 of these).

I think I'd favor us not having this and instead always using `messages.*` in the code. I think that helps make it easier to tell where the interface comes from (in our code vs a library). Not every library deals with its interfaces in this way so that idea doesn't really apply in other places.

Fine to leave as is for this PR, but I'll maybe create an issue for us to try to come to an agreement on if we want to do this going forward or not and migrate code to whichever pattern we choose

Reply:
Yeah that makes sense. We could prolly use `eslint-plugin-import` to enforce whatever we settle on too.